File: 01_Array_Hashings/01_Contains_Dupliates/217_Contains_Duplicates.java
import java.util.HashSet;
import java.util.Set;

class Solution {
    public boolean hasDuplicate(int[] nums) {
        Set<Integer> nums_set = new HashSet<>();

        for (int num : nums){
            if (nums_set.contains(num)){
                return true;
            }
            nums_set.add(num);
        }

        return false;
    }
}

File: 01_Array_Hashings/01_Contains_Dupliates/217_Contains_Duplicates.cpp
#include <vector>
#include <unordered_set>

class Solution {
public:
    bool hasDuplicate(std::vector<int>& nums) {
        std::unordered_set<int> nums_set;
        for (int num : nums){
            if (nums_set.count(num)) {
                return true;
            }
            nums_set.insert(num);
        }
        return false;
    }
};